packages/fast-tooling-react/src/css-editor/color/color.props.ts

```diff
@@ -0,0 +1,23 @@
+import { ManagedClasses } from "@microsoft/fast-jss-manager-react";
+import { CSSColorClassNameContract } from "./color.style";
+
+export interface CSSColorValues {
+    color?: string;
+}
+
+export interface CSSColorUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+export interface CSSColorHandledProps
+    extends ManagedClasses<CSSColorClassNameContract> {
+    /**
+     * The data
```

Comment:
Whats the purpose for making this an object type? Should this simply accept "color"?

Reply:
we did that elsewhere (ie height).

Reply:
for clarity, any one of these components could be editing one or multiple CSS properties, so we are keeping the same API even though there may only be a single property being updated.